app/components/ReturnToTop/ReturnToTop.tsx

```diff
@@ -0,0 +1,58 @@
+'use client';
+
+import { useEffect, useState, type FC } from 'react';
+import { ArrowUpDoticon } from 'doticons/32';
```

Comment:
let's use the other arrow

```suggestion
import { ArrowUpDoticon } from 'doticons/16';
```